### ISSUE TITLE
查看文档中的voicelist，好像角色都是外国人。在哪能找到，中文角色的id列表？

### ISSUE THREAD

**Issue by eleven-monkey** [netease-youdao/EmotiVoice#99]
查看文档中的[voicelist](https://github.com/netease-youdao/EmotiVoice/wiki/%F0%9F%98%8A-voice-wiki-page)，好像角色都是外国人。虽然也能用来读中文文本，但是还是听得出带有外国口音。不过，我看介绍收费api的文档提到是有中文角色的，比如：shudingli、youxiaofu等等。请问，在哪能找到原生的中文角色id列表？或者给出几个中文母语角色的数字ID,谢谢。

**Comment by syq163** [comment]
Yes, the speakers in the [voicelist](https://github.com/netease-youdao/EmotiVoice/wiki/%F0%9F%98%8A-voice-wiki-page) are English natives. While they can speak Chinese, it may not sound completely natural. For speaker IDs with more natural Chinese voices, I would recommend 65, 92, 984, 985, 1088, and 9000.
If you have the option to use [The EmotiVoice HTTP API](https://github.com/netease-youdao/EmotiVoice/wiki/HTTP-API), there are many more voices with native Chinese accents that you can choose from. Additionally, you can contact [Zhiyun](https://ai.youdao.com/) for further information regarding this matter.

**Comment by eleven-monkey** [comment]
谢谢回复，如果能在这个开源项目中多提供几个母语中文角色，会让这个项目更受欢迎，更有影响力。

**Comment by syq163** [comment]
> 谢谢回复，如果能在这个开源项目中多提供几个母语中文角色，会让这个项目更受欢迎，更有影响力。

Thank you for your suggestion! The lack of native Chinese speakers is mainly due to licensing limitations. Therefore, we encourage you to clone your own voices using your own data. Additionally, we are actively working on improving the quality of the pretrained model.

**Comment by iamchenxin** [comment]
Is it because there is no open source Chinese voice library?
